client/src/panels/panel_declarations/finances/vote_stat/vote_stat_text.yaml

```diff
@@ -109,7 +106,7 @@ gov_in_year_estimates_split_text:
 dept_in_year_estimates_split_text:
   transform: [handlebars,markdown]
   en: |
-   As of the {{est_in_year}} {{last_estimates}}, **{{subject.name}}**'s planned budgetary expenditures of **{{fmt_compact1_written dept_tabled_est_in_year_estimates}}** were presented in the 
+   As of the {{est_in_year}} {{last_estimates}}, **{{subject.name}}**'s {{gl_tt "authorized" "AUTH"}} spendings of **{{fmt_compact1_written dept_tabled_est_in_year_estimates}}** were presented in the 
```

Comment:
"..., {{name}} has been authorized to spend a total of {{value}} through the following... documents:"

I think adding "documents" or "mechanisms" after "adjustments and transfers" is useful to clarify what we're referring to below. "mechanism" is our more internal term, "document" may be more plain language but I'm not sure if it's the best wording choice for accuracy? Would need a second opinion.